proto-lens-tutorial/coffee-order/README.md

```diff
@@ -91,78 +82,57 @@ data Coffee'CoffeeType = Coffee'Americano !Americano
-         Lens.Labels.HasLens' f Coffee "maybe'cappuccino"
-           (Prelude.Maybe Cappuccino)
-         where
--- definition
-
-instance Prelude.Functor f =>
-         Lens.Labels.HasLens' f Coffee "cappuccino" (Cappuccino)
-         where
--- definition
-
-instance Prelude.Functor f =>
-         Lens.Labels.HasLens' f Coffee "maybe'mocha" (Prelude.Maybe Mocha)
-         where
--- definition
-
-instance Prelude.Functor f =>
-         Lens.Labels.HasLens' f Coffee "mocha" (Mocha)
-         where
--- definition
+instance Lens.Labels.HasLens' Coffee "maybe'coffeeType"
+            (Prelude.Maybe Coffee'CoffeeType)
+    where ...
+
+
+instance Lens.Labels.HasLens' Coffee "maybe'americano"
+             (Prelude.Maybe Americano)
+    where ...
+
+instance Lens.Labels.HasLens' Coffee "americano" (Americano)
+    where ...
+
+instance Lens.Labels.HasLens' Coffee "maybe'latte" (Prelude.Maybe Latte)
+    where ...
+
+instance Lens.Labels.HasLens' Coffee "latte" (Latte)
+    where ...
+
+instance Lens.Labels.HasLens' Coffee "maybe'flatWhite"
+             (Prelude.Maybe FlatWhite)
+    where ...
+
+instance Lens.Labels.HasLens' Coffee "flatWhite" (FlatWhite)
+    where ...
+
+instance Lens.Labels.HasLens' Coffee "maybe'cappuccino"
+             (Prelude.Maybe Cappuccino)
+    where ...
+
+instance Lens.Labels.HasLens' Coffee "cappuccino" (Cappuccino)
+    where ...
+
+instance Lens.Labels.HasLens' Coffee "maybe'mocha" (Prelude.Maybe Mocha)
+    where ...
+
+instance Lens.Labels.HasLens' Coffee "mocha" (Mocha)
+    where ...
 ```
 
 We have a bunch of lenses (or prisms) to access the data relating to `Coffee`. We can inspect the possibility of a `Coffee'CoffeeType` value in a `Coffee` value by using `maybe'coffeeType`. If we want to focus on a certain coffee within that sum type, for example `Mocha` we can use `maybe'mocha`. The word "inspect" is key here, we are viewing the possibility of values. So we if we used `mocha` for viewing a `Mocha` in `Coffee` we would end up with unknown data because it could be something entirely different, such as a `FlatWhite`.<br>
 
 What we can use them for is setting values! When we are defining the value we will know (and have to know) what type our `CoffeeType` will be. Thus we can do the following, `defMessage & mocha .~ defMessage` where the second `defMessage` is secretly our empty `Mocha` value.<br>
 
-As of commit `87efe00a0fa16b657493ba4043abd014d5c04cd3`, `Prism'` functions are also available for use on `Coffee'Type` sum. The `Prism'` functions are of of the naming convention `_Coffee'Americano`, `_Coffee'Latte`, etc. There is more information on `Prism'`s [here](../../docs/tutorial.md#oneof-generation)
+As of `proto-lens-protoc-0.3.0.0` and later, `Prism'` functions are also available for use on `Coffee'Type` sum. The `Prism'` functions are of of the naming convention `_Coffee'Americano`, `_Coffee'Latte`, etc. There is more information on `Prism'`s [here](../../docs/tutorial.md#oneof-generation)
```

Comment:
Do you mean linking to `#oneof` rather than `#oneof-generation`?  That didn't work for me when I tried it out, possibly because it's linking to a different file.

Reply:
"functions are *of of* the naming convention"